ui/app/templates/vault/cluster/access/oidc.hbs

```diff
@@ -25,14 +25,12 @@
       </DocLink>
     </p>
     {{#if this.isCta}}
-      <button
+      <Hds::Button
+        @text="Create your first app"
         type="submit"
-        class="button is-primary"
         {{on "click" (transition-to "vault.cluster.access.oidc.clients.create")}}
         data-test-oidc-configure
-      >
-        Create your first app
-      </button>
+      />
```

Comment:
Same comment here about removing this type and changing the on click to  a `@route` arg